src/agent/src/policy.rs

```diff
@@ -216,8 +220,17 @@ impl AgentPolicy {
         Ok(())
     }
 
-    /// Ask regorus if an API call should be allowed or not.
     async fn allow_request(&mut self, ep: &str, ep_input: &str) -> Result<(bool, String)> {
+        match self.version {
+            PolicyVersions::V1 => self.allow_request_v1(ep, ep_input).await,
+            PolicyVersions::V2 => {
```

Comment:
Are we introducing a behavior here that rejects anything that is any newer version? Someone can pass in higher than V2.